assets/pages/bundles-landing/components/Bundles.jsx

```diff
@@ -256,6 +290,20 @@ function ContributionBundle(props: PropTypes) {
         onNumberInputKeyPress={onClick}
         {...props}
       />
+
+      {props.contribType === 'ONE_OFF' && !!contribAttrs.paypalCta &&
+        <PayPalContributionButton
+          buttonText={contribAttrs.paypalCta.text}
+          amount={Number(props.contribAmount.oneOff.value)}
+          referrerAcquisitionData={props.referrerAcquisitionData}
+          isoCountry={props.isoCountry}
+          /* eslint no-alert: "off" */
```

Comment:
I would restrict the scope of this comment by using:

`// eslint-disable-next-line no-alert: "off"`